.gitmodules

```diff
@@ -1,6 +1,6 @@
 [submodule "detox/ios/COSTouchVisualizer"]
 	path = detox/ios/COSTouchVisualizer
-	url = https://github.com/joeblau/COSTouchVisualizer
+	url = https://github.com/wix-incubator/COSTouchVisualizer
```

Comment:
FYI @d4vidi, this repo isn't active for years.. There were a few fixes that needed to be made. Of course I made a PR to the original repo.